lib/modules/datasource/deb/index.spec.ts

```diff
@@ -1,14 +1,14 @@
 import { createReadStream } from 'fs';
-import { copyFile, stat } from 'fs-extra';
+import { copyFile } from 'fs/promises';
```

Comment:
```suggestion
import { copyFile } from 'fs-extra';
```

use `fs-extra` if possible